components/motor/motor.go

```diff
@@ -200,3 +199,22 @@ func CheckSpeed(rpm, max float64) (string, error) {
 		return "", nil
 	}
 }
+
+// CheckRevolutions checks if the input revolutions is non-zero
+func CheckRevolutions(revs float64) error {
+	if revs == 0 {
+		return NewZeroRevsError()
+	}
+	return nil
+}
+
+// GetDirection returns the direction based on the rpm and revolutions
+func GetDirection(rpm, revolutions float64) float64 {
+	dir := 1.0
+	if rpm*revolutions == 0.0 {
+		dir = 0.0
+	} else if rpm*revolutions < 0.0 {
+		dir = -1.0
+	}
+	return dir
+}
```

Comment:
Please rename this because `GetDirection` implies I'm getting the motor's direction as it is is moving, not getting the commanded Direction from the rpm and revolution requests.